robocop/checkers/errors.py

```diff
@@ -60,14 +60,15 @@
     "0404": Rule(
         rule_id="0404",
         name="variables-import-with-args",
-        msg="Robot and YAML variable files do not take arguments",
+        msg="YAML variable files do not take arguments",
         severity=RuleSeverity.ERROR,
+        version="<5.0",
```

Comment:
By doing that, this rule will be disabled for 5 and 6+ RF versions. I think it should be still enabled, but only report on arguments supplied to yaml file.

Reply:
You're totally right. I was in a hurry and I should mark it as WIP, because I didn't even make the tests work.
Thanks for noting that. I will update the code.

Reply:
I think very slowly lately but I just revisited the issue and I think we should not even use the version condition here. The rule should always report if the YAML file has arguments, because it's not possible for any RF version. The 5+ versions only add a possibility to import the variable file using module name. Am I right?

Reply:
I agree, there is no need to limit the version. We just need to raport only on yaml variablw file with arguments (and not on python variable file like it is now).